db/migrate/20151114125209_create_friendly_id_slugs.rb

```diff
@@ -0,0 +1,19 @@
+class CreateFriendlyIdSlugs < ActiveRecord::Migration
+  def change
+    create_table :friendly_id_slugs do |t|
```

Comment:
I used friendly some years ago, and I think Friendly will use the field `sluggable_type` added here to map your object. So, for each model of your project you just extend Friendly (`extend FriendlyId`) and set the field you need to slug (`friendly_id :name, use: :slugged`), for our Books model for example:
`class Book < ActiveRecord::Base`
     `    extend FriendlyId`
     `    friendly_id :title, use: :slugged`
`end`   

https://github.com/norman/friendly_id#rails-quickstart

Have this module loaded is usefull in cases when we have only the book id:
`book = Book.friendly.find(params[:id])`
`book.slug #=> "slug-book"`